site/en/docs/devtools/rendering/index.md

```diff
@@ -0,0 +1,356 @@
+
+The [`forced-colors` CSS media feature][5] indicates if the user agent enabled a forced colors mode. An example of a forced colors mode is Windows High Contrast.
+
+To emulate this condition:
+
+1. [Open the **Rendering** tab](#open-rendering).
+1. Under the **Emulate CSS media feature `forced-colors`**, select one of the following from the dropdown list:
+
+   - No emulation
+   - `forced-colors:active`
+   - `forced-colors:none`
+
+With `forced-colors:active` emulated:
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/ezZ8wZmzGYWvciitKSso.png", alt="forced-colors:active", width="800", height="398" %}
+
+## Emulate CSS media feature `prefers-contrast`
+
+The [`prefers-contrast` CSS media feature][6] indicates if the user requested the web content to present with a higher, lower, or specific contrast value.
+
+To emulate this condition:
+
+1. [Open the **Rendering** tab](#open-rendering).
+1. Under the **Emulate CSS media feature `prefers-contrast`**, select one of the following from the dropdown list:
+
+   - No emulation
+   - `prefers-contrast:more`
+   - `prefers-contrast:less`
+   - `prefers-contrast:custom`
+
+## Emulate CSS media feature `prefers-reduced-motion`
+
+The [`prefers-reduced-motion` CSS media feature][8] indicates if the user has requested to minimize the amount of motion on a page.
+
+To emulate this condition:
+
+1. [Open the **Rendering** tab](#open-rendering) on the [demo page](https://prefers-reduced-motion.glitch.me/) and try scrolling to see various animations.
+1. Under the **Emulate CSS media feature `prefers-reduced-motion`**, select `prefers-reduced-motion:reduce`.
+1. Try scrolling again.
+
+## Emulate CSS media feature `color-gamut`
+
+The [`color-gamut` CSS media feature][9] indicates which range of colors the user agent and the output device support.
+
+To emulate this condition:
+
+1. [Open the **Rendering** tab](#open-rendering).
+1. Under the **Emulate CSS media feature `color-gamut`**, select one of the following from the dropdown list:
+
+   - No emulation
+   - `color-gamut:srgb`—approximately [sRGB](https://en.wikipedia.org/wiki/SRGB) gamut or more
+   - `color-gamut:p3`—approximately the gamut specified in [Display P3 Color Space](https://www.color.org/chardata/rgb/DisplayP3.xalter) or more
+   - `color-gamut:rec2020`—approximately the gamut specified in [Rec. 2020](https://en.wikipedia.org/wiki/Rec._2020) or more
+
+## Emulate vision deficiencies
+
+Everyone should be able to access and enjoy the web. [Google is committed to making that a reality](https://www.google.com/accessibility/).
+
+With Chrome DevTools, you can see how people with vision deficiencies see your site, so you can make it better for them.
+
+To emulate vision deficiencies:
+
+1. [Open the **Rendering** tab](#open-rendering).
+1. Under the **Emulate CSS media feature `color-gamut`**, select one of the following from the dropdown list:
+
+   - No emulation
+   - Blurred vision
+   - Protanopia—can't perceive red and green colors
```

Comment:
Inconsistent options styles. Do we not bold / bold the options or use back tick?